demo-ehr-app/src/pages/MedicalImaging.tsx

```diff
@@ -0,0 +1,460 @@
+  const [selectedArea, setSelectedArea] = useState("");
+  const [selectedDate, setSelectedDate] = useState<Dayjs | null>(null);
+  const [selectedSlots, setSelectedSlots] = useState("");
+  const [selectedTreatingDisease, setSelectedTreatingDisease] = useState("");
+
+  const [isTestTypeChanged, setIsTestTypeChanged] = useState(false);
+  const [isSelectedAreaChanged, setIsSelectedAreaChanged] = useState(false);
+  const [isSelectedDateChanged, setIsSelectedDateChanged] = useState(false);
+  const [isSelectedSlotsChanged, setIsSelectedSlotsChanged] = useState(false);
+  const [
+    isSelectedTreatingDiseaseChanged,
+    setIsSelectedTreatingDiseaseChanged,
+  ] = useState(false);
+
+  const [enableSubmitButton, setEnableSubmitButton] = useState(false);
+  const [enableNextStep, setEnableNextStep] = useState(false);
+  const [isCrdCheckFinished, setCrdCheckFinished] = useState(false);
+  const [cdsCards, setCdsCards] = useState<CdsCard[]>([]);
+
+  const validateError = (
+    <>
+      <Alert severity="error" sx={{ marginLeft: 1 }}>
+        This is an mandotary field.
+      </Alert>
+    </>
+  );
+
+  const [patientId] = useState("PA2347");
+  const [practionerId] = useState("PT567498");
+
+  const handleChangeTestType = (event: SelectChangeEvent) => {
+    setSelectedTestType(event.target.value);
+    setCrdCheckFinished(false);
+  };
+
+  const handleChangeArea = (event: SelectChangeEvent) => {
+    setSelectedArea(event.target.value);
+    setCrdCheckFinished(false);
+  };
+
+  const handleDateChange = (
+    date: Dayjs | null,
+    _context: PickerChangeHandlerContext<DateValidationError>
+  ) => {
+    setSelectedDate(date);
+    setCrdCheckFinished(false);
+  };
+
+  const handleChangeSlots = (event: SelectChangeEvent) => {
+    setSelectedSlots(event.target.value);
+    setCrdCheckFinished(false);
+  };
+
+  const handleChangeTreatingDisease = (event: SelectChangeEvent) => {
+    setSelectedTreatingDisease(event.target.value);
+    setCrdCheckFinished(false);
+  };
+
+  const handleOnBlurTestType = () => {
+    setIsTestTypeChanged(true);
+  };
+
+  const handleOnBlurArea = () => {
+    setIsSelectedAreaChanged(true);
+  };
+
+  const handleOnBlurDate = () => {
+    setIsSelectedDateChanged(true);
+  };
+
+  const handleOnBlurSlots = () => {
+    setIsSelectedSlotsChanged(true);
+  };
+
+  const handleOnTreatingDisease = () => {
+    setIsSelectedTreatingDiseaseChanged(true);
+  };
+
+  const handleScheduleClick = () => {
+    setEnableNextStep(true);
+  };
+
+  const handleCrdCheck = () => {
+    const a = ORDER_SIGN_CDS_REQUEST2(
+      patientId,
+      practionerId,
+      selectedDate?.format("YYYY-MM-DD"),
+      selectedSlots
+    );
+
+    setCdsCards([]);
+    setEnableSubmitButton(false);
+    axios
+      .post<CdsResponse>(
+        "https://c32618cf-389d-44f1-93ee-b67a3468aae3-dev.e1-us-east-azure.choreoapis.dev/cmsdemosetups/cds-server/v1.0/cds-services/radiology-order",
```

Comment:
shall we take this to a config file?

Reply:
and check the other places as well

Reply:
Fixed. https://github.com/wso2/solutions-healthcare-demos/pull/2/commits/afd92c12223917bcd39b551d212f4129aae9b4bc